What you will learn partial

Woulei opened the pull request:
Issue #20

Final styling and content needs to be edited later when full page can be viewed.

arnoFleming commented:
Where do you render that `wywl` partial? and also, what does it mean? rename it to express intent!

Woulei commented:
Replaced it to different folder and renamed it. Also edited the classes for better implementation.
@arnoFleming Could you review again? ( @leefreemanxyz aswell maybe?)

arnoFleming commented:
after renaming the file (again :D) it LGTM

leefreemanxyz commented on app/views/layouts/events/_what_you_will_learn.html.erb:
```diff
@@ -0,0 +1,27 @@
+      <h3>Professional developers technologies</h3>
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-4">
```
I think we might want these to be col-md-4 rather than col-lg-4. Once you are on a browser window smaller than 1200px, I think this will all stack to single column.

Woulei commented:
@leefreemanxyz Solved, please approve
